qemu/tests/openflow_acl_test.py

```diff
@@ -276,9 +277,14 @@ def setup_service(setup_target):
         error_context.context("Set up %s service in %s"
                               % (setup_params.get("service"), setup_target),
                               logging.info)
+        setup_func(setup_cmd, timeout=setup_timeout)
+        if params.get("copy_ftp_site") and setup_target != "localhost":
+            ftp_site = os.path.join(data_dir.get_deps_dir(), params.get("copy_ftp_site"))
+            ftp_dir = params.get("ftp_dir")
+            setup_vm.copy_files_to(ftp_site, ftp_dir)
+
```

Comment:
@xiywang Just double confirm, is this what we expect to exchange the order of `setup_cmd` and `prepare_cmd`?

Reply:
@vivianQizhu Yes, that's OK.